pkg/configflags/helpers.go

```diff
@@ -0,0 +1,43 @@
+package configflags
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// ArgsWithPrefix filters arguments by prefix and collects values.
+func ArgsWithPrefix(args map[string][]string, prefix string) map[string][]string {
+	filtered := map[string][]string{}
+	for key, slice := range args {
+		if !strings.HasPrefix("audit-", key) {
```

Comment:
You want to filter by prefix, not `audit-`

Reply:
fixed